hedera-mirror-web3/src/main/java/com/hedera/services/utils/EntityIdUtils.java

```diff
@@ -130,6 +131,14 @@ public static TokenID tokenIdFromEvmAddress(final Address address) {
         return tokenIdFromEvmAddress(address.toArrayUnsafe());
     }
 
+    public static boolean isOfEvmAddressSize(final ByteString alias) {
+        return alias.size() == EVM_ADDRESS_SIZE;
+    }
+
+    public static boolean isOfEcdsaAddressSize(final ByteString alias) {
```

Comment:
This is for a publicKey right? If so
```suggestion
    public static boolean isOfEcdsaPublicKeySize(final ByteString alias) {
```

Reply:
Renamed